new/LIT5843ShortPoemMary.xml

```diff
@@ -53,11 +50,19 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     </teiHeader>
     <text>
         <body>
-            <div type="bibliography">
+             <div type="bibliography">
                 <listRelation>
                     <relation name="lawd:hasAttestation" active="LIT5843ShortPoemMary" passive="Neukirchen005"/>
                 </listRelation>
-            </div><!---->
+                <div type="textpart" subtype="chapter" n="7" corresp="#ms_i3" xml:id="chapter7" xml:lang="gez">
```

Comment:
corresp="**Neukirchen005**#ms_i3"

Reply:
check also the rest in the line, why n="7"? is it also copied from the MS record?

Reply:
Done.